guides/v3.18.0/components/built-in-components.md

```diff
@@ -72,13 +72,13 @@ The following event types are supported (dasherized format):
 * `@key-up`
 
 
-More [events types](https://api.emberjs.com/ember/release/classes/Component#event-names) are also supported but these events need to be written in camelCase format, such `mouseEnter`. Note, there are events of the same type in both the list above and linked. Event names listed above must be dasherized. Additional work is performed on these events.
+More [events types](https://api.emberjs.com/ember/3.18/classes/Component#event-names) are also supported but these events need to be written in camelCase format, such `mouseEnter`. Note, there are events of the same type in both the list above and linked. Event names listed above must be dasherized. Additional work is performed on these events.
```

Comment:
The `#event-names` anchor doesn't exist. I wasn't sure what'd be a good substitute.

Reply:
I see what you mean, although I think we should still link to /release/. 

I think the missing anchor tag is for the "Event Handler Methods" - is there a way to add the anchor in to the correct place?

Reply:
Hmm, I think the version would be `3.18` according to what the issue asked for (point API links in 3.18 Guides to 3.18 API Guides?).

I think you're right that the anchor can point to `https://api.emberjs.com/ember/3.18/classes/Component#event-handler-methods` instead, since that section lists examples of events.

Reply:
I found that, prior to `v3.10`, the API Guides used to have a section called "Event Names." Since `v3.10`, that section was renamed to "Event Handler Methods."

- https://api.emberjs.com/ember/3.9/classes/Component#event-names
- https://api.emberjs.com/ember/3.10/classes/Component#event-handler-methods

I pushed a commit to update the links for v3.7 - 3.19.